2_delivery_care.do

```diff
@@ -77,6 +77,7 @@ order *,sequential  //make sure variables are in order.
 	*c_caesarean: Last birth in last 2 years delivered through caesarean                    
 	clonevar c_caesarean = m17
 	replace c_caesarean = . if c_caesarean == 9
+	replace c_caesarean = . if m15 == 11 | m15 == 12 | m15 == 96
```

Comment:
Glad that you code basing on your understanding of this variable. Sorry that I forgot to explain this: in the DHS raw data, m17 is coded 0 when m15=11,12,96, that is to regard mother didn't receive caesarean if she delivered at home. It makes sense to us, so we follow their way of treatment. Therefore, there is no need to code `c_caesarean =.` when `m15 == 11|m15 == 12|m15 == 96`.
Instead, I would add:
`replace c_caesarean = . if inlist(m15,.,99,98)`